dotnet/TF2SA.Data/Models/JoinedStats.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TF2SA.Data.Models
+{
+    public class JoinedStats
+    {
+      
+        public ulong SteamId { get; set; }
+        public string? PlayerName { get; set; }
+        public uint PlayerStatsId { get; set; }
+        public uint GameId { get; set; }
+        public byte TeamId { get; set; }
+        public uint? DamageTaken { get; set; }
+        public uint? HealsReceived { get; set; }
+        public uint? MedkitsHp { get; set; }
+        public byte? Airshots { get; set; }
+        public byte? Headshots { get; set; }
+        public byte? Backstabs { get; set; }
+        public byte? Drops { get; set; }
+        public uint? Heals { get; set; }
+        public byte? Ubers { get; set; }
+        public uint ClassStatsId { get; set; }
+        public byte ClassId { get; set; }
+        public ushort? Playtime { get; set; }
+        public byte? Kills { get; set; }
+        public byte? Assists { get; set; }
+        public byte? Deaths { get; set; }
+        public uint? Damage { get; set; }
+
+        public JoinedStats(ulong SteamId, string? PlayerName, uint PlayerStatsId,
```

Comment:
stylistically, I think it's neater if you just newline after every single parameter ie:
public JoinedStats(
    ulong SteamId,
    string? PlayerName,